tkey/src/main/java/com/web3auth/tkey/ThresholdKey/Modules/TSSModule.java

```diff
@@ -0,0 +1,472 @@
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+
+class GetTSSPubKeyResult {
+    public static class Point {
+        public String x;
+        public String y;
+
+        public String toFullAddr() {
+            return "04" + x + y;
+        }
+
+        public Point(String x, String y) {
+            this.x = x;
+            this.y = y;
+        }
+    }
+
+    public Point publicKey;
+    public List<BigInteger> nodeIndexes;
+
+    public GetTSSPubKeyResult(Point publicKey, List<BigInteger> nodeIndexes) {
+        this.publicKey = publicKey;
+        this.nodeIndexes = nodeIndexes;
+    }
+}
+
+public final class TSSModule {
+    public static String curveN = "fffffffffffffffffffffffffffffffebaaedce6af48a03bbfd25e8cd0364141";
+
+    private TSSModule() {
+        //Utility class
+    }
+
+    private static native String jniTSSModuleGetTSSPublicKey(ThresholdKey thresholdKey, RuntimeError error);
+    
+    private static native String jniTSSModuleGetAllTSSTags(ThresholdKey thresholdKey, RuntimeError error);
+    
+    private static native String jniTSSTagFactorPub(ThresholdKey thresholdKey, RuntimeError error);
+    
+    private static native String jniGetExtendedVerifier(ThresholdKey thresholdKey, RuntimeError error);
+
+    private static native void jniTSSModuleSetTSSTag(ThresholdKey thresholdKey, String tssTag, RuntimeError error);
+    
+    private static native String jniTSSModuleGetTSSTag(ThresholdKey thresholdKey, RuntimeError error);
+    
+    private static native void jniTSSModuleCreateTaggedTSSShare(ThresholdKey thresholdKey, String deviceTssShare, String factorPub, int deviceTssIndex, String curveN, RuntimeError error);
+    
+    private static native String jniTSSModuleGetTSSShare(ThresholdKey thresholdKey, String factorKey, int threshold, String curveN, RuntimeError error);
+    
+    private static native int jniGetTSSNonce(ThresholdKey thresholdKey, String tssTag, RuntimeError error);
+
+    private static native void jniCopyFactorPub(ThresholdKey thresholdKey, String newFactorPub, int newTssIndex, String factorPub, String curveN, RuntimeError error);
+    
+    private static native void jniGenerateTSSShare(ThresholdKey thresholdKey, String inputTssShare, int inputTssIndex, int newTssIndex, String newFactorPub,  String selectedServers, String authSignatures, String curveN, RuntimeError error);
+
+    private static native void jniDeleteTSSShare(ThresholdKey thresholdKey, String inputTssShare, int inputTssIndex, String factorPub,  String selectedServers, String authSignatures, String curveN, RuntimeError error);
+
+    public static native void jniThresholdKeyServiceProviderAssignPublicKey(ThresholdKey thresholdKey, String tss_tag, String tss_nonce, String tss_public_key, RuntimeError error);
+
+    public static String getTSSPubKey(ThresholdKey thresholdKey) throws RuntimeError {
+        RuntimeError error = new RuntimeError();
```

Comment:
First the tss tag must be set, before retrieving the tss public key.

See here:
https://github.com/tkey/tkey-swift/pull/36/files#diff-673a6b022fa5809c3dae1324992e2a1dbe5eec52b8fbbcb16651cac42c8e586bR113-R124